src/compiler/scala/tools/cmd/CommandLineParser.scala

```diff
@@ -35,7 +35,15 @@ object CommandLineParser {
   }
   private object DoubleQuoted extends QuotedExtractor('"')
   private object SingleQuoted extends QuotedExtractor('\'')
-  private val Word = """(\S+)(.*)""".r
+  object Word {
+    private val regex = """(\S+)""".r
+    def unapply(s: String): Option[(String, String)] = {
+      regex.findPrefixOf(s) match {
+        case Some(prefix) => Some(prefix, s.substring(prefix.length, s.length()))
```

Comment:
laugh + confused emoji might be ambiguous. @ jenkins, this comment should not require human intervention, sorry, I mean intervention by a team developer. Jenkins, old chap, you are the humanist i.e. most human, not "human bigot". Can robots be bigoted or only roboted?